prisma/schema.prisma

```diff
@@ -5,6 +5,7 @@ datasource db {
 
 generator client {
   provider = "prisma-client-js"
+  previewFeatures = ["referentialIntegrity"]
```

Comment:
oh what's this?

Reply:
I think it comes with `onDelete: Cascade`

> In [3.0.1](https://github.com/prisma/prisma/releases/tag/3.0.1) and later it is possible to do cascading deletes using [Referential actions](https://www.prisma.io/docs/concepts/components/prisma-schema/relations/referential-actions). Be sure to follow the [upgrade guide](https://www.prisma.io/docs/guides/upgrade-guides/upgrading-versions/upgrading-to-prisma-3/referential-actions) before using this feature.